packages/ag-charts-website/package.json

```diff
@@ -59,7 +59,6 @@
     "glob": "8.0.3",
     "dotenv": "10.0.0",
     "commander": "2.20.3",
-    "mime": "^4.0.4",
```

Comment:
Why are you deleting it here? Aren't you using it in the website?

Reply:
The tsx files were causing an issue with this package, so I moved back to the original implementation